blocks/tabs/tabs.css

```diff
@@ -1,4 +1,4 @@
-.block.tabs .tabs-list {
+.block.media.tabs .tabs-list {
```

Comment:
Nit: 
@ravverma since media and ipo are in tabs block should this selection be 
`block.tabs.media`

and same for ipo as well ?

Reply:
yes make sense. will change it.

Reply:
done